src/measure_theory/measure/haar_lebesgue.lean

```diff
@@ -369,6 +370,55 @@ calc μ (affine_map.homothety x r '' s) = μ ((λ y, y + x) '' (r • ((λ y, y
 ... = ennreal.of_real (abs (r ^ (finrank ℝ E))) * μ s :
   by simp only [image_add_right, measure_preimage_add_right, add_haar_smul]
 
+lemma add_haar_univ [nontrivial E] : μ (univ : set E) = ∞ :=
```

Comment:
It seems that this is true for any noncompact locally compact group (not sure if we need the group to be locally compact but I failed to quickly prove it without this assumption). Indeed, take two compact nhds of `1` such that `K₁ * K₁⁻¹ ⊆ K₂`. Then take a sequence of points $g_n$ such that $g_n\notin\Union_{k<n} g_k⋅K₂`. Then `g_n ⋅ K₁` are pairwise disjoint, hence total measure is greater than `∞ * μ K₁ = ∞`.

Reply:
It took me longer than I expected to get it right, but I have now done the general version for noncompact groups.